genon/preprocessor/facade/convert_processor.py

```
@@ -3355,8 +3377,33 @@ async def _process_xlsx(self, request: Request, file_path: str, **kwargs: dict):
+                "1", f"동일 doc_type 에 tabular custom_fields 설정이 여러 개입니다: {runtime_doc_type}"
+            )
+        if matching_mappers:
+            _log.info(f"[DocumentProcessor] xlsx tabular custom_fields 처리(doc_type={runtime_doc_type}): {file_path}")
+            try:
+                vectors = build_tabular_custom_fields_vectors(
+                    file_path, matching_mappers[0], runtime_doc_type,
+                    header_row=self._xlsx_cfg["header_row"],
+                    multi_table=self._xlsx_cfg["multi_table"],
+                )
+            except (FileNotFoundError, TypeError, ValueError) as exc:
+                raise GenosServiceException("1", str(exc))
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)